autogen.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 
-autoreconf -fiv
+autoreconf -fiv --include tests/cbtest
```

Comment:
@mynktl we need to change cbtest to cstor here as well

Reply:
ok